### ISSUE TITLE
Fix Commands: "lategame" and "midgame"

### ISSUE THREAD

**Issue by Judgy53** [harbingerofme/DebugToolkit#143]
**Which Command**
`lategame` and `midgame`

**What should it do?**
Since 3.8.1 changed `random_items`'s format, these 2 macros don't give items anymore. Adding `Tier1Def,Tier2Def,Tier3Def` as a parameter does technically work, but wouldn't fully replicate the old behavior (no weighting).

**Comment by SChinchi** [comment]
The command change was made to add support for custom tiers. Yes, it does break compatibility with how the the macro commands used to behave, but they still give items.

Ideally we need to add some weighting option to make the command complete and between `random_items {count} tier1:weight1,tier2:weight2,etc` and `random_items {count} tier1,tier2 weight1,weight2` I favour the latter because it's less hassle to work with "all" tiers. Still feels a bit clunky and a better approach might exist.

But coming back to the issue, if we want to have the exact same behavior as in the past, it's still possible by using the HG random items method that the old command used to call internally.

**Comment by Judgy53** [comment]
>  Yes, it does break compatibility [...] but they still give items.

I tried both macros several times but never received any items (mod version 3.8.2). The [macro](https://github.com/harbingerofme/DebugToolkit/blob/e014cd06b906fbf8c2773a6e5d3e4ae14f92b129/Code/DT-Commands/Macros.cs#L33) sends `user.username` as the second argument, which fails to be parsed as a tier and [short circuit](https://github.com/harbingerofme/DebugToolkit/blob/e014cd06b906fbf8c2773a6e5d3e4ae14f92b129/Code/DT-Commands/Items.cs#L200) the whole command. 

That's why I suggested adding the parameter `Tier1Def,Tier2Def,Tier3Def`. Another possibility would be to try to parse the second argument as a user before trying to parse it as a tier. Not sure what would be better.
